common/encoding.h

```diff
@@ -40,47 +40,47 @@ namespace Common {
  */
 
 /**
- * A class, that allows conversion between different text encoding,
- * the encodings available depend on the current backend and if the
+ * A class that allows conversion between different text encoding.
+ * The encodings available depend on the current backend and whether
  * ScummVM is compiled with or without iconv.
  */
 class Encoding {
 	public:
 		/**
-		 * Constructs everything needed for the conversion between 2 encodings
+		 * Construct everything needed for the conversion between two encodings
 		 * and saves the values for future use.
 		 *
-		 * @param to Name of the encoding the strings will be converted to
-		 * @param from Name of the encoding the strings will be converted from
+		 * @param to   Name of the encoding the strings will be converted to.
+		 * @param from Name of the encoding the strings will be converted from.
 		 */
 		Encoding(const String &to, const String &from);
 		~Encoding() {};
 
 		/**
-		 * Converts string between encodings. The resulting string is ended by 
-		 * a character with value 0 (C-like ending for 1 byte per character
-		 * encodings, 2 zero bytes for UTF-16, 4 zero bytes for UTF-32)
+		 * Convert a string between encodings.
 		 *
-		 * The result has to be freed after use.
+         * The resulting string is ended by a character with value 0 
+		 * (C-like ending for 1 byte per character encodings, 2 zero bytes for UTF-16,
+		 * 4 zero bytes for UTF-32). The result must be freed after usage.
```

Comment:
It might be worth mentioning that the result must be freed using `free()` and not `delete[]`.